datastore/setup.py

```diff
@@ -50,13 +50,13 @@
 
 
 REQUIREMENTS = [
-    'google-cloud-core >= 0.21.0, < 0.22dev',
+    'google-cloud-core >= 0.22.1, < 0.23dev',
     'grpcio >= 1.0.0, < 2.0dev',
```

Comment:
Will you update this to `>= 1.0.2rc0` (or `1.0.2`)? I've done that for `grpc-google-cloud-logging-v2`.